### ISSUE TITLE
Restart Addon by Automation?

### ISSUE THREAD

**Issue by scrichab** [Akulatraxas/ha-modbusproxy#19]
Hi,
thank you for your Addon 
is it possible to restart Addon by Automation?

**Comment by daemonenstall** [comment]
Yes, that's possible. You have to use the restart add-on from supervisor service.
![IMG_20240420_113400](https://github.com/Akulatraxas/ha-modbusproxy/assets/69319145/d48840ec-6d8e-42d9-b399-d55e734587e2)
